src/components/Details/carsDetails.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { useAuthHeader } from 'react-auth-kit';
+import { useParams, useDispatch } from 'react-router-dom';
+// import { useParams, useNavigate } from 'react-router-dom';
+import { AiOutlineRight } from 'react-icons/ai';
+import Loader from '../Loader/Loader';
+import { fetchCars } from '../../redux/cars/cars';
+
+const DetailsCarScreen = () => {
+  const [loading, setLoading] = useState(true);
+  const dispatch = useDispatch();
+  const authHeader = useAuthHeader();
+  const { id } = useParams();
+  const carDetails = useSelector((state) => state.cars);
+  const cars = carDetails;
+  const carss = cars?.find((c) => c.id === parseInt(id, 10));
+
+  useEffect(() => {
+    dispatch(fetchCars(authHeader()));
+    setTimeout(() => {
+      setLoading(false);
+    }, 3800);
+  }, [dispatch]);
+
+  if (loading) {
+    return <Loader />;
+  }
+
+  if (!cars) {
+    return (
+      <>
+        <h2>Sorry</h2>
+        <p>
+          There is no car in Garage
+        </p>
+
+      </>
+    );
+  }
+
+  return (
+    <>
+      {carss && (
```

Comment:
Kindly update the design of the details page to match the behance design better🙏

Your design:
<img width="587" alt="image" src="https://user-images.githubusercontent.com/37116322/230637142-d41b0033-abe8-4699-b9a3-a1b3955dc78e.png">


Behance design:
<img width="587" alt="image" src="https://user-images.githubusercontent.com/37116322/230637218-a4249174-40b3-4e60-aa09-eba557b724f8.png">

A few things to note:
- Please include more details for the car. The car has a type, model, year, and price_per_day
- Kindly increase the right padding of the details. Its a bit too close to the edge
- Kindly raise the car details to be higher up on the page. Not on the same line with the image
- Kindly increase the spacing between `other details` and `discover more models`
- Kindly add the reserve button on the bottom right of the page. Its okay if it doesn't work for now, at least let it be there
- Kindly add the back button on the bottom left of the page. it was there before but you removed it